contracts/libraries/SnappBaseCore.sol

```diff
@@ -63,6 +63,14 @@ library SnappBaseCore {
     /**
      * Public View Methods
      */
+    function depositBatchSize() public pure returns (uint8) {
```

Comment:
DEPOSIT_BATCH_SIZE is already declared as public. I guess you could just use the official read function instead of coding your own read function

Reply:
Unfortunately reading from a Library is not as easy as it looks... Would you care to try and demonstrate how this might work?